### ISSUE TITLE
The plugin is throwing an error

### ISSUE THREAD

**Issue by larrynalzaro** [kasperlegarth/instastory.js#23]
Hi there,

For some reason, the plugin is throwing this error:

`Uncaught TypeError: data.graphql is undefined`

This occurred today. I wonder if Instagram has stopped supporting graphql query?

![instastory_error](https://user-images.githubusercontent.com/75696897/101532786-e27a6500-3949-11eb-907f-4a10a8197298.png)

**Comment by kasperlegarth** [comment]
Hi @larrynalzaro,

This is not an official API endpoint by Instagram, so this is just an assumption, but I think there is an IP limitation on the endpoint. I have tried to explain here: #21

**TLDR:** It's temporary due to development behavior where you refresh a lot.

I hope this clears things :)

**Comment by larrynalzaro** [comment]
Thanks for the explanation. Great plugin, btw.


Cheers.

**Comment by vbroskas** [comment]
> Hi @larrynalzaro,
> 
> This is not an official API endpoint by Instagram, so this is just an assumption, but I think there is an IP limitation on the endpoint. I have tried to explain here: #21
> 
> **TLDR:** It's temporary due to development behavior where you refresh a lot.
> 
> I hope this clears things :)

I'm getting the same error on firefox and a slightly different one in chrome:

`Uncaught TypeError: Cannot read property 'user' of undefined
    at Object.success (VM408 instastory.js:159)
    at i (jquery-1.12.0.min.js:2)
    at Object.fireWith [as resolveWith] (jquery-1.12.0.min.js:2)
    at y (jquery-1.12.0.min.js:4)
    at XMLHttpRequest.c (jquery-1.12.0.min.js:4)`

**EDIT** just saw what you wrote about blocking the users IP. All devices I was checking with were under 1 network. Looks like it's displaying correctly on devices on other IPs. Thanks!

**Comment by kasperlegarth** [comment]
@vbroskas I'm glad you figured it out :)

**Comment by kasperlegarth** [comment]
@larrynalzaro Thanks for the kind words :)
